src/main/resources/allowed-labels.properties

```diff
@@ -12,6 +12,7 @@ android
 api-plugin
 aws
 azure
+bamboo
```

Comment:
I only found one Jenkins plugin that seems to match this tag:
* [Bamboo pipeline step](https://plugins.jenkins.io/pipeline-bamboo/)

Since Bamboo is a continuous integration server, I would not expect to have many Jenkins plugins related to Bamboo.  

I think it is premature to whitelist a `bamboo` label until we know there are multiple plugins that would use the label